components/PageNotFound/index.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { ReactNode } from "react";
+import styles from "./PageNotFound.module.css";
+
+import NextLink from "next/link";
+import Image from "next/image";
+
+import skoolLogo from "@/public/skool.svg";
+
+const PageNotFound = () => {
+  return (
```

Comment:
One little thing, that is for SEO and UX. Each page should have a separate `<Head>` component in NextJS, basically this is the NextJS way of declaring separate <head> for each document, since we are switching page, the <head> should be different for different page. 

For example, one thing could be the `title`, which will be shown on the tab in your chrome when you hit a 404 Error. You could check out the Skool version, and you will see how does that boost the user experience.

Take a look at this [page](https://nextjs.org/docs/pages/api-reference/components/head) in the Next documentation.